templates/accounts/registration.html

```diff
@@ -7,6 +7,23 @@
     $(function () {
     	$("#id_username").focus()
     });
+    $(document).ready(function(){
+      if (!($('#username_check')).length) {
+        $('#id_username').parent().append('<div id="username_check" style="color: #bd2d33;font-style: italic;"></div>');
```

Comment:
It might be cleaner to work with http://api.jquery.com/toggle/ rather than setting the HTML content...